### ISSUE TITLE
About the problem formulation

### ISSUE THREAD

**Issue by zzt007** [Flawless1202/Non-AR-Spatial-Temporal-Transformer#5]
Hi, thanks for your great job!
There is a problem bother me a lot , which about the paper's problem formulation.
![image](https://github.com/user-attachments/assets/0537e9c8-13f8-4798-ba4b-b9d02e4aff59)

N_0 means the total count of tracked objects in the sequence, and the T_h means the observation horizon.
However, during the sequence length, the N_0 may change, and here is appears to be a fixed value?
In other words, the lifetime of the tracked object may be less than the sequence length.
So I don't know how to due with these 'variables' in my model. 
Looking forward to your early reply!
